platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,21 +42,8 @@
     <bean id="security" class="org.codice.ddf.security.common.Security"
           factory-method="getInstance"/>
 
-    <!-- Base URI for DDF home -->
-    <bean id="ddfHomeFile" class="java.io.File">
-        <argument value="${ddf.home}"/>
-    </bean>
-
-    <bean id="ddfHomeUri" class="java.net.URI" factory-ref="ddfHomeFile" factory-method="toURI"/>
-
-    <!-- Default Directory Export Path -->
-    <bean id="defaultExportDirectoryUri" class="java.nio.file.Uri" factory-ref="ddfHomeUri"
-          factory-method="resolve">
+    <bean id="defaultExportDirectoryPath" class="java.io.File">
         <argument value="${ddf.home}/etc/exported"/>
```

Comment:
If we are going to do this we can create a bean using the `AbsolutePathResolver` which will be cleaner and consistent. This is across all of our blueprints, if we're going to fix it then we should do it everywhere.